components/gitpod-cli/cmd/rebuild.go

```diff
@@ -241,6 +241,11 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		return err
 	}
 
+	err = os.MkdirAll("/workspace/.docker-root-debug", 0710)
```

Comment:
@iQQBot I think we should think about long term approach, we will need to move there VS Code user data and JB config/cache folders.

I thought that in regular workspace we should do something like:
```
/workspace/.gitpod/root/.docker
/workspace/.gitpod/root/.vscode-remote
/workspace/.gitpod/root/.cache
/workspace/.gitpod/root/.config
```

And then for the debug workspace use something like `/workspace/.gitpod/debug-root`?

We don't need to change for the regular workspace for now. But for the debug workspace we could do it clean? i.e. start now `/workspace/.gitpod/debug-root/.docker`. 

I wonder whether it also applies to `/workspace/.gitpod` folder generally? i.e why would we shate gitpod tasks between them?

Maybe instead it should move folders under `/workspace/.gitpod` and in the debug mount `/workspace/.gitpod-debug:/workspace/.gitpod`

WDYT? Should we talk sync?

Reply:
@iQQBot we maybe should consider to keep debug workspace data somewhere under /var/tmp? Because of recent perf concerns

Reply:
> @iQQBot we maybe should consider to keep debug workspace data somewhere under /var/tmp? Because of recent perf concerns

Pudong said it is impossible, because linux allows only 2 layers, and we already have it for root fs

For the structure we agreed on:
```
/workspace/.gitpod-debug/.docker-root -> /workspace/.docker-root
/workspace/.gitpod-debug/.gitpod -> /workspace/.gitpod
/workspace/.gitpod-debug/.vscode-remote -> /workspace/.vscode-remote
/workspace/.gitpod-debug/.cache -> /workspace/.cache
/workspace/.gitpod-debug/.config -> /workspace/.config
```